README_API.md

```diff
@@ -17,11 +19,16 @@ These URIs notify observers if on content changes (e.g., a new `TrackPoint` was
 
 Limitations:
 * only tracks selected by the user are exposed,
-* `Waypoints` are not exposed,
+* `Waypoint`s are not exposed,
 * access is only granted temporarily and automatically revoked (see `FLAG_GRANT_READ_URI_PERMISSION`),
-* no write access is possible, and
-* neither track recordings can be started, paused, or stopped.
+* no write access is possible
+
+The Data API can also be started automatically by leveraging Extra fields in the explicit Intent for the Start/Stop API.
```

Comment:
DashboardAPI?

Reply:
Actually, I used "Data API" because it's in the title of this file. And "Start/Stop API" because it's new and is just called "API" in README.md